.goreleaser.docker.yaml

```diff
@@ -17,10 +17,7 @@ env:
   - 'CGO_ENABLED=0'
   - 'GO111MODULE=on'
   - 'GOPROXY=https://proxy.golang.org,direct'
-  # Allow override the container registry.
-  # Default to the JVS CI container registry.
-  - REGISTRY={{ if index .Env "CONTAINER_REGISTRY"  }}{{ .Env.CONTAINER_REGISTRY }}{{ else }}us-docker.pkg.dev/jvs-ci-test/docker-images{{ end }}
-
```

Comment:
This is intentional. Why remove?

Reply:
This is there for the us-docker.pkg.dev/jvs-ci-test/docker-images fall back I assume. I removed this thinking we may only need REGISTRY, and pass it everytime we run goreleaser, this is what we did for pmap. Any concerns with this change?